py-scripts/lf_interop_video_streaming.py

```diff
@@ -281,8 +311,8 @@ def start_specific(self,cx_start_list):
                 while self.local_realm.json_get("/cx/" + i).get(i).get('state') != 'Run':
                     continue
         except Exception as e:
-            pass
-
+            logger.info(f"Exception occured: {e}")
+        logging.info("Test started at : {0} ".format(datetime.now().strftime("%Y-%m-%d %H:%M:%S")))
     # def create_generic_endp(self,query_resources,os_types_dict):
```

Comment:
I recognize this was present before, but let's remove this unused code if no longer relevant in a future commit. No change required.